deployment/charts/templates/deploy.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.app.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-service-monitor
```

Comment:
Please test this by doing 2 deployments in the same namespace. My feeling is that the second one is not going to work because it has the same object name

Reply:
I hardcoded the release name in the deployment script `service-deploy.sh`:
![image](https://github.com/FuelLabs/fuel-block-committer/assets/794823/b7094cb9-816d-45d9-aee2-24dd3e8e441a)

Due to this, subsequent workflows just update a singular deployment within the namespace. Should I try installing the charts under different release names or do you think this approach of hardcoding the release name via helm is sufficient?

Reply:
fixed by using the chart release name as an additional selector